Mobile Buy SDK/Mobile Buy SDK/Data/BUYClient+Checkout.m

```diff
@@ -152,6 +153,15 @@ - (NSOperation *)getCompletionStatusOfCheckoutWithToken:(NSString *)token start:
 	}];
 }
 
+- (NSOperation *)pollCompletionStatusAndGetCheckoutWithToken:(NSString *)token start:(BOOL)start completion:(BUYDataCheckoutBlock)block
```

Comment:
OK. We will wait and see what the community has to say and if they ask for it or something similar.